openpgp/packet/signature.go

```diff
@@ -452,7 +452,7 @@ func subpacketsLength(subpackets []outputSubpacket, hashed bool) (length int) {
 	for _, subpacket := range subpackets {
 		if subpacket.hashed == hashed {
 			length += subpacketLengthLength(len(subpacket.contents) + 1)
-			length += 1 // type byte
+			length ++ // type byte
```

Comment:
I think the reason they wrote it the way it was is to emphasize the fact that it's 1 byte. (I.e. we're adding an amount of bytes to the length that happens to be 1, rather than incrementing it, specifically.)

Reply:
Okay, why not; reverting this.